core/src/main/resources/error/error-classes.json

```diff
@@ -270,6 +270,11 @@
           "The <exprName> must not be null"
         ]
       },
+      "UNEXPECTED_PARAMETER_CONSTRAINT" : {
```

Comment:
How about `PARAMETER_CONSTRAINT_VIOLATION`

Reply:
fine to me